docs/user/security/authentication/index.asciidoc

```diff
@@ -65,6 +65,10 @@ image::user/security/images/kibana-login.png["Login Selector UI"]
 
 For more information, refer to <<authentication-security-settings, authentication security settings>>.
 
+TIP: If you have multiple authentication providers configured, you can use the `auth_provider_hint` URL query parameter to create a deep
+link to any provider and bypass the Login Selector UI. Using the `kibana.yml` above as an example, you can add `?auth_provider_hint=basic1`
```

Comment:
__optional nit:__ we may want to also add that it would work even if this particular option isn't displayed in Login Selector UI.

Reply:
Ah, it's good to know that. I think in the interest of brevity I will leave it as-is, but if we get questions about that down the road we can always add it :+1: